styles/typography.scss

```diff
@@ -0,0 +1,70 @@
+:root {
+  --fs-900: clamp(5rem, 8vw + 1rem, 9.375rem);
+  --fs-800: 3.5rem;
+  --fs-700: 1.5rem;
+  --fs-600: 1rem;
+  --fs-500: 1rem;
+  --fs-400: 0.9375rem;
+  --fs-300: 1rem;
+  --fs-200: 0.875rem;
+
+  --ff-serif: 'Bellefair', serif;
+  --ff-sans-cond: 'Barlow Condensed', sans-serif;
+  --ff-sans-normal: 'Barlow', sans-serif;
```

Comment:
I'm assuming you have some solution for this (because you told me about it before we left for Sanian ❤️) but this Pull Request breaks the artist-page on mobile:

<img width="363" alt="Screenshot 2023-01-28 at 15 34 51" src="https://user-images.githubusercontent.com/7456867/215272368-9faa8d40-bd3e-4921-8fc1-4590ee3685a0.png">
<img width="367" alt="Screenshot 2023-01-28 at 15 35 10" src="https://user-images.githubusercontent.com/7456867/215272372-57d8fd2a-5c8c-429f-b7c4-ae6eddda72da.png">


Would love if we could fix this before pulling this into the main branch <3

Reply:
I fix dis